package.json

```diff
@@ -40,7 +41,8 @@
     "lint:fix": "yarn run lint --fix",
     "format": "prettier --write '**/*.{json,md,scss,yaml,yml}'",
     "format:check": "prettier --check '**/*.{json,md,scss,yaml,yml}'",
-    "update-api": "./scripts/update-api.sh ./src/common/api/swagger.yaml ./src/common/api/types.ts"
+    "update-api": "./scripts/update-api.sh ./src/common/api/swagger.yaml ./src/common/api/types.ts",
+    "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c ./scripts/i18next-parser.config.js  && ./scripts/clean-translation-files.sh"
```

Comment:
@ammont82 as a suggestion, I think this script would read better if the entry point would be in a file named `./scripts/i18n.sh`, there you can have the call to `i18next` and also include the content of `./scripts/clean-translation-files.sh` in the same file.  
Have a look at `./scripts/update-api.sh` as an example